package.json

```
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/alastria/alastria-identity-example#readme",
   "dependencies": {
-    "alastria-identity-lib": "github:alastria/alastria-identity-lib#master",
+    "alastria-identity-lib": "github:alastria/alastria-identity-lib#feature/support-upgradeable-contracts",
```

```suggestion
    "alastria-identity-lib": "github:alastria/alastria-identity-lib#develop",
```
We should update the develop branch in _alastria-identity-lib_ in order to use the existing branch

---

Ok i was wrong sorry, what we really need is to update the `ContractInfo.md` file in the `alastria-identity` respository with the new smart contracts addresses. Once that is done, the library should be able to get the updated information